evaluator.expression/src/main/java/org/opencds/cqf/cql/evaluator/expression/ExpressionEvaluator.java

```diff
@@ -231,6 +231,7 @@ private String constructCqlLibrary(String expression, List<Pair<String, String>>
         cql = sb.toString();
 
         logger.debug(cql);
+        System.out.println(sb.toString());
```

Comment:
We should never write directly to std out. The one exception is in the cli project that's meant to interact with the user terminal directly. This line needs to be deleted.

If you want debug output configure the logger accordingly.

Reply:
Sysout Removed